### PR TITLE
Improvements to operators documentation

### PR DESCRIPTION
**Pull request by nkuba**
This PR improves existing Keep ECDSA client documentation and adds a doc about tBTC liquidation recovery for the client operators.

Depends on https://github.com/keep-network/keep-ecdsa/pull/893

### CODE REVIEW

File: docs/run-keep-ecdsa.adoc [hunk 1]
```diff
@@ -130,280 +131,185 @@ Application configurations are stored in a `.toml` file and passed to the applic
...
-    MaxFeePerVByte = 75
-    # The bitcoin chain that you want to connect to. default: "mainnet"
-    # allowed values: ["mainnet", "regtest", "simnet", "testnet3"]
-    BitcoinChainName = "mainnet"
-    # An endpoint pointing to a running electrs
-    # (https://github.com/Blockstream/electrs) service. The officially hosted
-    # one works, but you can run your own node!
-    # To explicitly disable automatic broadcasting, set this value to the empty string "".
-    ElectrsURL = "https://blockstream.info/api/"
+include::../configs/config.toml.SAMPLE[]
```
Comment (beaushinkle) [inline docs/run-keep-ecdsa.adoc:136]: that's awesome

Comment (nkuba) [reply docs/run-keep-ecdsa.adoc:136]: I had wanted to do it for a long time, and suddenly it turned out to be a one-liner 😆

File: docs/tbtc-liquidation-recovery.adoc [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+== Background
+
+.As described in xref:rfc/rfc-2.adoc[RFC 2]:
+[quote]
+____
+include::rfc/rfc-2.adoc[tag=background]
+____
+
+Keep ECDSA client implements a mechanism mentioned above to recover BTC after
```
Comment (beaushinkle) [inline docs/tbtc-liquidation-recovery.adoc:26]:
```suggestion
Keep ECDSA client implements the mechanism mentioned above to recover BTC after
```

File: docs/tbtc-liquidation-recovery.adoc [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+a tBTC Deposit got liquidated and split the amount among the Keep signers holding
+the BTC.
+
+== Configuration
+
+To enable the tBTC Liquidation Recovery mechanism you have to update TOML configuration
```
Comment (beaushinkle) [inline docs/tbtc-liquidation-recovery.adoc:32]:
```suggestion
To enable the tBTC Liquidation Recovery mechanism, update the TOML configuration
```

File: docs/tbtc-liquidation-recovery.adoc [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+file with details of destination (`BeneficiaryAddress`) to which Bitcoin funds should
```
Comment (beaushinkle) [inline docs/tbtc-liquidation-recovery.adoc:33]:
```suggestion
file with a destination (`BeneficiaryAddress`) to which Bitcoin funds should
```

File: docs/tbtc-liquidation-recovery.adoc [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+Bitcoin address for each Bitcoin transaction.
+
+Addresses are resolved starting from `0` index. The index is incremented by 1 in
+case of the address being used already.
+
+The highest used index for the extended public keys will be stored in the client's
+local storage directory under `bitcoin/derivation_indexes/<EXTENDED_PUBLIC_KEY_ID>/<INDEX>`
+(e.g. `bitcoin/derivation_indexes/xpub_b1wex5vq/3`).
+
+Additionally to the local storage check the client will reach Bitcoin API to
```
Comment (beaushinkle) [inline docs/tbtc-liquidation-recovery.adoc:93]:
```suggestion
The client will additionally reach the Bitcoin API to
```

File: docs/tbtc-liquidation-recovery.adoc [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+check if there are any existing transactions for the given address.
+
+If the Bitcoin connection is missing the client will check the address usage
+based on the local storage only.
+
+=== Bitcoin Chain Connectivity
+
+Connectivity to Bitcoin chain API is used for liquidation recovery handling but is not required for the process to complete.
```
Comment (beaushinkle) [inline docs/tbtc-liquidation-recovery.adoc:101]: Here we call it the Bitcoin chain API, but above we cal it just the Bitcoin API. I think we can standardize one one for consistency. (My vote is Bitcoin API but my preference isn't strong)

**Review by beaushinkle — CHANGES_REQUESTED** [review]

File: docs/run-keep-ecdsa.adoc (hunk 1 above)
Comment (beaushinkle) [reply docs/run-keep-ecdsa.adoc:136]: looks funky when viewed in github:

image

Comment (nkuba) [reply docs/run-keep-ecdsa.adoc:136]: Yeah, this is a known GitHub limitation https://github.com/github/markup/issues/1095
What we can do is publish HTML generated docs https://github.com/keep-network/keep-ecdsa/pull/893 it looks fine there https://docs.keep.network/ecdsa/docs-workflow/run-keep-ecdsa.html.

File: docs/tbtc-liquidation-recovery.adoc (hunk 6 above)
Comment (nkuba) [reply docs/tbtc-liquidation-recovery.adoc]: 4d445deb03bdd43954ecc1262a61675a1d0f35f9

File: docs/tbtc-liquidation-recovery.adoc (hunk 5 above)
Comment (nkuba) [reply docs/tbtc-liquidation-recovery.adoc]: 4d445deb03bdd43954ecc1262a61675a1d0f35f9

File: docs/tbtc-liquidation-recovery.adoc (hunk 4 above)
Comment (nkuba) [reply docs/tbtc-liquidation-recovery.adoc]: 4d445deb03bdd43954ecc1262a61675a1d0f35f9

File: docs/tbtc-liquidation-recovery.adoc (hunk 3 above)
Comment (nkuba) [reply docs/tbtc-liquidation-recovery.adoc]: 4d445deb03bdd43954ecc1262a61675a1d0f35f9

File: docs/tbtc-liquidation-recovery.adoc (hunk 2 above)
Comment (nkuba) [reply docs/tbtc-liquidation-recovery.adoc]: 4d445deb03bdd43954ecc1262a61675a1d0f35f9

**Review by beaushinkle — APPROVED** [review]

**Pull request merged**